pkg/sql/contention/event_store.go

```diff
@@ -348,6 +353,27 @@ func (s *eventStore) resolutionIntervalWithJitter() time.Duration {
 	return jitteredInterval
 }
 
+func logResolvedEvents(ctx context.Context, events []contentionpb.ExtendedContentionEvent) {
```

Comment:
another thing to discuss - should the fingerprints be in the hex format (so that they can be used directly to query `statement_statistics` / `transaction_statistics`) or be in the int format (as used in the URL for SQL activity pages) ?